src/shared/components/navbar.tsx

```diff
@@ -1,69 +1,126 @@
-import { Component, ChangeEvent, linkEvent } from "inferno";
+import { ChangeEvent, linkEvent } from "inferno";
 import { Link } from "inferno-router";
-import { LinkLine } from "./link-line";
-import { Icon } from "./icon";
-import { i18n, languages } from "../i18next";
+import { Icon, IconSize } from "./icon";
+import { i18n, LANGUAGES } from "../i18next";
+import classNames from "classnames";
+import { SELECT_CLASSES } from "./common";
 
-export class Navbar extends Component<any, any> {
-  constructor(props: any, context: any) {
-    super(props, context);
-  }
+const NavLink = ({ content }) => <li className="text-gray-400">{content}</li>;
 
-  handleLanguageChange(_: any, event: ChangeEvent<HTMLSelectElement>) {
-    location.href = `/?lang=${event.target.value}`;
-  }
+const NavLinks = () => (
+  <>
+    <NavLink
+      content={
+        <Link onClick={() => closeNavbarDropdown()} to="/instances">
+          {i18n.t("join")}
+        </Link>
+      }
+    />
+    <NavLink
+      content={
+        <Link onClick={() => closeNavbarDropdown()} to="/news">
+          {i18n.t("news")}
+        </Link>
+      }
+    />
+    <NavLink
+      content={
+        <Link onClick={() => closeNavbarDropdown()} to="/apps">
+          {i18n.t("apps")}
+        </Link>
+      }
+    />
+    <NavLink
+      content={
+        <Link onClick={() => closeNavbarDropdown()} to="/donate">
+          {i18n.t("donate")}
+        </Link>
+      }
+    />
+    <NavLink
+      content={
+        <a onClick={() => closeNavbarDropdown()} href={`/docs/index.html`}>
```

Comment:
If you're using a string literal for the `href`, you don't need the curly braces.